src/components/ChangePlan/Checkout/Step/Step.js

```diff
@@ -2,48 +2,46 @@ import React from 'react';
 import { useIntl } from 'react-intl';
 
 /* TODO: These styles will be removed with the correct classes. Currently Gus is working with the layout. */
-const containerStyle = {
-  paddingLeft: '20px',
-  paddingTop: '20px',
-  paddingRight: '20px',
-  border: '2px solid #eaeaea',
-  borderRadius: '3px',
-  marginTop: '20px',
-};
-
 const displaFlexAndjustifyContentSpaceBetweenStyle = {
   display: 'flex',
   justifyContent: 'space-between',
 };
 
-const upperCaseStyle = {
+const titleStyle = {
```

Comment:
@jhoffman-ms Why do we still have inline styles here? Now with the new styles applied this should not be here, or be made into a styled component.

Reply:
Because Gus didn't work with this part yet. For it reason I maintain the inline style.

Reply:
Perhaps add a TODO comment later then.

Reply:
Currently in the file I added the following comment:

``` 
/* TODO: These styles will be removed with the correct classes. Currently Gus is working with the layout. */
```